.github/workflows/goreleasemac.yml

```diff
@@ -0,0 +1,31 @@
+name: goreleasermac
```

Comment:
why this separate pipeline? can't one just extend "make release-dry-run" and "make release" used in https://github.com/crypto-org-chain/chain-main/blob/master/.github/workflows/goreleaser.yml ?

Reply:
troian/golang-cross doesn't support arm64+ darwin, yet
```
    goos:
      - darwin
    goarch:
      - arm64
      
```

Reply:
error is  like this
![image](https://user-images.githubusercontent.com/51560997/111094460-51351300-8576-11eb-8095-42db88810936.png)

Reply:
It supports Linux-arm64 though -- so perhaps, the easiest solution may be just to extend the current goreleaser config for Linux-arm64 (so one can deploy e.g. on AWS Graviton https://pages.awscloud.com/rs/112-TZM-766/images/2020_0501-CMP_Slide-Deck.pdf ) ... for M1, there are no inexpensive "servers" one can easily deploy to, so not having official binaries isn't a big problem -- for development, one can build them themselves or use Rosetta.
Later, when the "Darwin-arm64" is supported on troian/golang-cross, we can just extend the config

Reply:
adding to current yml is possible, i'll include it.

Reply:
yes, i'll add linux arm64, that would be done very easily.
apple m1 binary is tested successfully, i'll submit to this pr